src/components/inline-inputs/inline-inputs.component.js

```diff
@@ -32,17 +33,33 @@ const InlineInputs = (props) => {
     gutter
   } = props;
 
+  const labelId = guid();
```

Comment:
This will lead to a new id on each render

Reply:
fixed 07f8227